mysite/urls.py

```diff
@@ -14,9 +14,11 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
+from django.conf.urls import include
 from django.urls import path, include
```

Comment:
You import include twice. Is this necessary?

Reply:
Done.